standalone/docker-compose.yml

```diff
@@ -1,19 +1,22 @@
 version: '3.7'
 
 services:
-  powersimdata:
-    container_name: powersimdata
-    hostname: powersimdata
+  client:
+    container_name: client
+    hostname: client
     build:
-      context: ../../PowerSimData
-    image: ghcr.io/breakthrough-energy/powersimdata:latest
-    entrypoint: bash
+      context: ../../PostREISE
+    image: ghcr.io/breakthrough-energy/postreise:latest
     stdin_open: true # docker run -i
     tty: true        # docker run -t
+    working_dir: /plug
+    entrypoint: bash
     volumes:
       - scenario_data:/mnt/bes/pcm
       - ../rootdir/raw:/mnt/bes/pcm/raw
-      - ./:/PowerSimData/plug
+      - ./:/plug
```

Comment:
I don't understand the voules' mount. Where will be `scenario_data`? the relative path are relative to what folder?

Reply:
If left side is a relative path, it's from the current directory on the host machine, and right side is path in container. In the top line, `scenario_data` is a named volume, so it's managed by docker and not something a user would access (they could if they really want, since it still has to be stored on disk).

Reply:
Thanks